README.md

```diff
@@ -33,7 +33,7 @@ Then, for multi-line scripts:
 Or, for single line commands:
 
 ```yaml
-  - run: msys2 uname -a
+  - run: msys2 -c 'uname -a'
```

Comment:
To summarize, I think the only required change is to rewrite this. For example:

~~~
```yaml
  - shell: msys2
    run: uname -a
```

It is also possible to execute specific commands from cmd/powershell scripts/snippets. In order to do so, `-c` is required:

```yaml
  - shell: powershell
    run: msys2 -c 'uname -a'
```

```yaml
  - shell: cmd
    run: msys2 -c 'uname -a'
```
~~~

Reply:
done. I also removed the reference the "multi-line scripts" one example above, since it is the same for "single lines".